src/GafferUIBindings/PlugAdderBinding.cpp

```diff
@@ -89,13 +92,71 @@ struct PlugMenuSlotCaller
 
 };
 
+struct PlugAdderWrapper : public PlugAdder, IECorePython::Wrapper< PlugAdder >
+{
+
+	PlugAdderWrapper(PyObject *self, StandardNodeGadget::Edge edge ) : PlugAdder( edge ), IECorePython::Wrapper<PlugAdder>( self, this )
+	{
+	}
+
+	bool acceptsPlug( const Gaffer::Plug *connectionEndPoint ) const
+	{
+		IECorePython::ScopedGILLock gilLock;
```

Comment:
This all works slightly differently when using the Wrapper classes I mentioned above. First off, you should call `isSubclassed()` and avoid the override lookup entirely when there isn't a pure-python subclass. This allows us to avoid a GIL lock and method lookup for instances which are created from Python, but not derived-from in Python We also must use `methodOverride()` rather than `get_override()`, and avoid the `override` class. There are examples throughout the codebase - [this one](https://github.com/GafferHQ/gaffer/blob/master/src/GafferSceneBindings/RenderBinding.cpp#L72) for instance.